### ISSUE TITLE
Go install complains

### ISSUE THREAD

**Issue by derekcollison** [mfridman/tparse#64]
Just tried to install from go per instructions.

`go install github.com/mfridman/tparse@latest`

Resulted in the following.

```
> go install github.com/mfridman/tparse@latest
go: downloading github.com/mfridman/tparse v0.10.2
go: downloading github.com/muesli/termenv v0.11.0
go: downloading github.com/charmbracelet/lipgloss v0.4.0
go: downloading github.com/olekukonko/tablewriter v0.0.5
go: downloading github.com/mattn/go-runewidth v0.0.13
go: downloading github.com/muesli/reflow v0.3.0
go: downloading github.com/rivo/uniseg v0.2.0
go: downloading golang.org/x/sys v0.0.0-20220513210249-45d2b4557a2a
# github.com/mfridman/tparse/internal/app
/Users/derek/Development/go/pkg/mod/github.com/mfridman/tparse@v0.10.2/internal/app/table_failed.go:52:18: undefined: strings.Cut
```

**Comment by mfridman** [comment]
Ouch, this didn't need to use `strings.Cut` (added in go1.18). Will fix this tonight so older `go` versions work as well.

**Comment by derekcollison** [comment]
Right now NATS server still on 1.17 but we will transition soon, thanks!

**Comment by mfridman** [comment]
This was fixed in [`v0.10.3`](https://github.com/mfridman/tparse/releases/tag/v0.10.3), and should work for older versions of Go. Thanks for reporting the issue.
